hack/tekton_in_kind.sh

```diff
@@ -37,10 +37,13 @@ while getopts ":c:p:t:d:" opt; do
     d )
       TEKTON_DASHBOARD_VERSION=$OPTARG
       ;;
+    k )
+      CONTAINER_RUNTIME="docker"
+      ;;
     \? )
       echo "Invalid option: $OPTARG" 1>&2
       echo 1>&2
-      echo "Usage:  tekton_in_kind.sh [-c cluster-name -p pipeline-version -t triggers-version -d dashboard-version]"
+      echo "Usage:  tekton_in_kind.sh [-c cluster-name -p pipeline-version -t triggers-version -d dashboard-version [-k]"
```

Comment:
should this be `-k container-runtime`?

Reply:
I defined it as a flag to avoid issues with spelling the name of the container runtime - so `-k` means docker, nothing means podman. I just realised though that I missed the "k" in the getopts :P